dapp/pages/_document.js

```diff
@@ -39,6 +39,7 @@ class MyDocument extends Document {
             type="text/javascript"
           ></script>
 
+          <meta name="image" content={"https://cmsmediaproduction.s3.amazonaws.com/meta_9121c5630d.jpeg"} />
```

Comment:
Why not have it as a local file as opposed to using a remote URL?

Reply:
i added a file and local path thanks. do you think that will work as is?